### DISCUSSION TITLE
v0.12.0

### DISCUSSION THREAD

**Discussion by anilbeesetti** [anilbeesetti/nextplayer#1034] [Announcements]
## New Features
* Add playback speed shortcuts and skip silence by @anilbeesetti in https://github.com/anilbeesetti/nextplayer/pull/991
* Add Swipe refresh to media by @anilbeesetti in https://github.com/anilbeesetti/nextplayer/pull/985
* Folder tree view mode by @anilbeesetti in https://github.com/anilbeesetti/nextplayer/pull/1015
* Add played progress indicator to video view by @anilbeesetti in https://github.com/anilbeesetti/nextplayer/pull/1028

## Improvements
* Charset detection by using full byte array by @anilbeesetti in https://github.com/anilbeesetti/nextplayer/pull/1008
* Add subtitle charset detection for network files by @anilbeesetti in https://github.com/anilbeesetti/nextplayer/pull/1012
* Update androidxMedia3 to v1.4.0 by @renovate in https://github.com/anilbeesetti/nextplayer/pull/1013

## Fixes
* Fixed Video Zoom not remembered between player sessions by @anilbeesetti in https://github.com/anilbeesetti/nextplayer/pull/1007
* Fixed portrait thumbnails are rotated by @anilbeesetti in https://github.com/anilbeesetti/nextlib/pull/82

<hr /><em>This discussion was created from the release <a href='https://github.com/anilbeesetti/nextplayer/releases/tag/v0.12.0'>v0.12.0</a>.</em>

**Comment by anilbeesetti** [comment]
Hey guys, If you find any issues in the new version let me know and i will fix them and i will promote this release to production after fixing issues in a week

**Comment by TidalMist** [comment]
Not shows videos from sd card on homepage. 0.11.1 shows
<img src="https://github.com/user-attachments/assets/e71ce8f9-b28f-4db1-80d6-80f26ee374b0" height="400"><img src="https://github.com/user-attachments/assets/f7612e0f-5ce7-4599-97e2-54bd6a5e03a3" height="400">

**Reply by anilbeesetti** [reply]
Need more info like may be share some screenshots and explain in more detail

**Reply by ghost** [reply]
this happens to me too. All videos on SD card aren't visible and not shown.

**Comment by Aryan-Chauhan** [comment]
Thanks for doing https://github.com/anilbeesetti/nextplayer/pull/1007

Can you add this when you time?
https://github.com/anilbeesetti/nextplayer/issues/981

**Reply by anilbeesetti** [reply]
Sure, will add this in the next minor release

**Comment by hrshtein** [comment]
Thanks for your work! 

Please also look for #954
